<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SKBio</class>
 <widget class="QMainWindow" name="SKBio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SKBio</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="principal">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>521</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>Principal</string>
     </attribute>
     <widget class="QComboBox" name="comboBoxPersonas">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>353</x>
        <y>130</y>
        <width>141</width>
        <height>27</height>
       </rect>
      </property>
      <property name="accessibleName">
       <string/>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="modelColumn">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>285</x>
        <y>136</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnGuardarAsado">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>50</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Crear</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>136</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Paga   $</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lblPaga">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>130</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tbAsado">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>331</width>
        <height>321</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Nombre</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Paga</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="btnAgregarPersona">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>250</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBorrarPersona">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>310</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Borrar</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxAsado">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>151</width>
        <height>27</height>
       </rect>
      </property>
      <property name="accessibleName">
       <string/>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="modelColumn">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtFecha">
      <property name="geometry">
       <rect>
        <x>192</x>
        <y>50</y>
        <width>121</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Direcci√≥n:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Fecha:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Totales</string>
     </attribute>
     <widget class="QTableWidget" name="tbTotales">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>121</y>
        <width>501</width>
        <height>391</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Nombre</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Pone</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Saca</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>20</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Monto por persona</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Monto Total</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblMontoTotal">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>------------------</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblMontoPersona">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>80</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>--------------------</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Historial</string>
     </attribute>
     <widget class="QTableWidget" name="tbHistorial">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>461</width>
        <height>451</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Direccion</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Fecha</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Monto Total</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Monto/persona</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Personas</string>
     </attribute>
     <widget class="QTableWidget" name="tbPersonas">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>101</y>
        <width>511</width>
        <height>411</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Nombre</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Direccion</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Telefono</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Deuda</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="btnGuardar">
      <property name="geometry">
       <rect>
        <x>397</x>
        <y>30</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
